File: pueblo-golf-tournament-api/Dto/Outgoing/LookupPlayerProfileResponseDto.cs
using pueblo_golf_tournament_api.Base;

namespace pueblo_golf_tournament_api.Dto.Outgoing
{
    public class LookupPlayerProfileResponseDto : BaseResponse
    {
        public PlayerProfile? Profile { get; set; }
    }

    public class PlayerProfile
    {
        public PersonDto? Person { get; set; }
        public PlayerDto? Player { get; set; }
    }

    
}

File: pueblo_golf_tournament_mobile/lib/dto/response/lookup-player-profile-response-dto.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:pueblo_golf_tournament_mobile/base/base-response.dart';
import 'package:pueblo_golf_tournament_mobile/dto/response/lookup-tournament-team-response-dto.dart';

part 'lookup-player-profile-response-dto.g.dart';

@JsonSerializable()
class LookupPlayeProfileResponseDto extends BaseResponse {
  final PlayerProfile? profile;

  factory LookupPlayeProfileResponseDto.fromJson(Map<String, dynamic> json) =>
      _$LookupPlayeProfileResponseDtoFromJson(json);

  LookupPlayeProfileResponseDto({required super.message, this.profile});
  Map<String, dynamic> toJson() => _$LookupPlayeProfileResponseDtoToJson(this);
}
